dbms/src/Storages/KVStore/Read/LearnerReadWorker.cpp

```diff
@@ -256,10 +260,18 @@ void LearnerReadWorker::recordReadIndexError(
                     region_id);
                 region_status = RegionException::RegionReadStatus::KEY_NOT_IN_REGION;
             }
+            else if (region_error.has_server_is_busy())
+            {
+                // 1. From TiKV
+                // 2. Read index request timeout
+                GET_METRIC(tiflash_raft_learner_read_failures_count, type_read_index_timeout).Increment();
+                LOG_DEBUG(log, "meet abnormal region error {}, [region_id={}]", resp.ShortDebugString(), region_id);
```

Comment:
nit: maybe we need INFO level when these region_error happens?

Reply:
No, this error is as expected. A read index request is safe to be responded with timeout.